esp/bindings/http/client/httpclient.ipp

```diff
@@ -100,6 +102,10 @@ private:
     StringAttr m_password;
     StringAttr m_realm;
     ISecureSocketContext *m_ssctx;
+    IPersistentHandler* m_persistentHandler;
```

Comment:
better to initialize all of these new member variables here, rather than in the constructor.